scripts/pipelines/acquia-pipelines.yml

```diff
@@ -18,7 +18,7 @@ events:
               - composer validate --no-check-all --ansi
               - composer install --ansi
               - source ${BLT_DIR}/scripts/pipelines/setup_env
-              # - nvm install v6.9.1
+              # - nvm install v6.9.1 --latest-npm
```

Comment:
are you updating node version or npm version ?

`nvm install lts/*` installs the latest lts for example

Reply:
oh, it's a comment :D doh!